docs/sections/user_guide/uw_yaml/rocoto_yaml.rst

```diff
@@ -14,21 +14,21 @@ Starting at the top level of the UW YAML config for Rocoto, there are several re
 
 .. code-block:: yaml
 
-  workflow:
-    attrs:
-      realtime: false
-      scheduler: slurm
-    cycledef:
-      - attrs:
-          activation_offset: -06:00
-          group: howdy
-        spec: 202209290000 202209300000 06:00:00
-    entities:
-      ACCOUNT: myaccount
-      FOO: test.log
-    log: /some/path/to/&FOO;
-    tasks:
-      ...
+   workflow:
+     attrs:
+       realtime: false
+       scheduler: slurm
+     cycledef:
+       - attrs:
+           activation_offset: -06:00
+           group: howdy
+         spec: 202209290000 202209300000 06:00:00
+     entities:
+       ACCOUNT: myaccount
+       FOO: test.log
+     log: /some/path/to/&FOO;
+     tasks:
+       ...
```

Comment:
I somehow missed (or maybe accidentally reversed when dealing with a merge conflict) updating the indentation of code blocks in this document. Consider hiding whitespace in this PR diff to focus on the more meaningful changes.